doc/use_apps/rpc_api.rst

```diff
@@ -2,12 +2,7 @@ RPC API
 =======
 
 Application RPCs are exposed under ``/app`` and require authentication with a valid user identity.
-This is a sample of what an application endpoint may look like, as generated by the :ref:`example C++ logging app <build_apps/example_cpp>`.
-
-.. openapi:: ../schemas/app_openapi.json
-   :examples:
-   :include:
-      /app/log/.*
+You can read what application endpoints may look like, as generated by the :ref:`example C++ logging app <build_apps/example_cpp>`, in :doc:`/build_apps/example_rpc_api`.
```

Comment:
Is app_openapi.json expanded there?

Reply:
Yes, my thinking is that the built-in parts are expanded here, and then the parts which are more of an example (aka `/app/log/`) are expanded in `example_rpc_api.rst`

Reply:
Ok!